extensions/datasource/deployment-spi/src/main/java/io/quarkus/datasource/deployment/spi/DevServicesDatasourceConfigurationHandlerBuildItem.java

```diff
@@ -85,17 +86,31 @@ private static String datasourceURLPropName(String dsName) {
     }
 
     public static DevServicesDatasourceConfigurationHandlerBuildItem reactive(String dbKind) {
+        return reactive(dbKind, new Function<String, String>() {
+            @Override
+            public String apply(String url) {
+                return url.replaceFirst("jdbc:", "vertx-reactive:");
```

Comment:
Are we sure a JDBC URL is always a valid reactive one except for the prefix? I'm not sure that's true.

Reply:
@tsegismont can you check? https://docs.microsoft.com/en-us/sql/connect/jdbc/understanding-ssl-support?view=sql-server-ver15#remarks does not say much about that.

Reply:
To be honest, the current code is already doing the replace.

Reply:
>Are we sure a JDBC URL is always a valid reactive one except for the prefix? I'm not sure that's true.

@gsmet Definitely not. But as @cescoffier noticed, that's what we do today and [until we get something better](https://github.com/eclipse-vertx/vertx-sql-client/issues/947), we can avoid errors for users of dev services by removing the `encrypt` param.